### DISCUSSION TITLE
I have a .fr domain from an external registrar (OVH), will I be able to use it in Vercel?

### DISCUSSION THREAD

**Discussion by honocoder** [vercel/community#2936] [Help] [answered]
### Summary

Hello,

I have a domain in .fr (https://fanhypnose.fr) from OVH that I would like to transfer to Vercel, but I'm not sure that I will be available to use it there.

The manipulation sounds easy, but here's the issue : I tried to search new domains in Vercel (https://vercel.com/domains?limit=24), but it seems like there are no .fr domains.

I don't particularly want to buy a new one but only to transfer the one I have, but as Vercel don't have .fr domains, what will happen at the renew time? Will I still have my fanhypnose.fr domain and will pay for it on Vercel, even if it seems like they don't have those domains?

Thanks for your help!

### Steps to Reproduce

To see that there are no .fr domains in Vercel :

- Go to [this site](https://vercel.com/domains?limit=24)
- Try searching for any .fr domain, like supercoolexample.fr
- There are no results, only .frl, which is not what I want

**Comment by amyegan** [comment, marked as answer]
We do not currently support .fr TLD directly. However, you can use `A` and `CNAME` records to point the domain from your third party registrar to your Vercel project. You can learn more about adding and configuring a custom domain here: https://vercel.com/docs/concepts/projects/domains/add-a-domain

**Reply by honocoder** [reply]
Hey @amyegan, thanks for your quick answer and for the guide you provided.
Have a great day
